cypress/e2e/user.cy.js

```diff
@@ -1,12 +1,59 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
 
+import SignInPageObject from '../support/pages/signIn.pageObject';
+import UserPageObject from '../support/pages/user.pageObject';
+import ArticlePageObject from '../support/pages/article.pageObject';
+import SettingsPageObject from '../support/pages/settings.pageObject';
+import SignUpPageObject from '../support/pages/signUp.pageObject';
+
+const signInPage = new SignInPageObject();
+const userPage = new UserPageObject();
+const articlePage = new ArticlePageObject();
+const settingsPage = new SettingsPageObject();
+const signUpPage = new SignUpPageObject();
+
 describe('User', () => {
-  before(() => {
+  let user;
+  let article;
 
+  before(() => {
+    cy.task('db:clear');
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+      cy.register(user.email, user.username, user.password);
+    });
+    cy.task('generateArticle').then((generateArticle) => {
+      article = generateArticle;
+    });
   });
 
-  it.skip('should be able to follow the another user', () => {
```

Comment:
1. why it is skipped? 
2. all test data (users, articles, etc. should be prepared via API in preconditions (before, beforeEach) 
3. there is no assertion in the end of test